### PR TITLE
Added VSCode Support

### PR DESCRIPTION
**Pull request by dellis1972**

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        },
+        {
+            "id": "target",
+            "type": "pickString",
+            "default": "Build",
+            "description": "Select MSBuild Target to run.",
+            "options": [
+                "Build",
+                "Install",
+                "SignAndroidPackage",
```
Comment (jonathanpeppers) [inline .vscode/tasks.json:64]: I don't think we need this one, as `dotnet build` will produce an `.apk` in .NET 6.

File: README.md [hunk 2]
```diff
@@ -121,6 +121,35 @@ that since it refers to the legacy one.
...
+code net6-mobile-samples.code-workspace
+```
+
+To build your application use open the Command Pallette and select 
+`Run Build Task`. Select `Build` and then the `Target` you want to run.
+Available targets are
+
+* `Build` : Builds the Project.
+* `Install` : Installs the Application on a Device or Emulator.
+* `SignAndroidPackage` : Builds the package for Publishing.
```
Comment (jonathanpeppers) [inline README.md:139]: So we can probably remove `SignAndroidPackage`, from this doc, too.

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "DebugAndroid",
```
Comment (jonathanpeppers) [inline .vscode/launch.json:8]: Is there a way to name this `Debug` and it prompts to pick the project / platform?

Comment (dellis1972) [reply .vscode/launch.json:8]: It already asks you for the Project. So I guess once we support iOS we can just add the projects. But yes I can do that

**Pull request merged**